libparsec/crates/client/tests/unit/workspace/stat_entry.rs

```diff
@@ -121,6 +121,161 @@ async fn stat_entry(#[values(true, false)] local_cache: bool, env: &TestbedEnv)
     );
 }
 
+async fn stat_entry_by_id_helper(
+    ops: &WorkspaceOps,
+    entry_id: VlobID,
+    kind: &'_ str,
+    expected_confinement_point: Option<VlobID>,
+) -> (EntryStat, Option<VlobID>) {
+    match kind {
+        "vanilla" => (
```

Comment:
In this context, vanilla means we care about the confinement point but we don't know it ?

Reply:
571c530cf9aacf14e2379652fc3b6187774559f0